src/components/application/ApplicationSubmitted.tsx

```diff
@@ -47,14 +50,17 @@ export const ApplicationSubmitted = ({ className }: { className?: string }) => {
           height={80}
           width={90}
         />
-        <h2 className="text-2xl font-semibold text-center">
-          Application Submitted!
-        </h2>
-        <p className="text-center">
-          You&apos;ll receive an email at{" "}
-          <span className="font-medium text-muted-foreground">
-            shaun@optimism.io
-          </span>{" "}
+        <h2 className="text-center">Application Submitted!</h2>
+        <p className="text-center text-lg font-semibold max-w-lg">
+          You&apos;ll receive an email{" "}
+          {email ? (
```

Comment:
oooh what if they don't have an email? maybe we can pop up the email modal

Reply:
i believe we're supposed to pop it on the previous step in the dashboard, but also you could just link straight here? hmmm good point let's add this to the follow-ups list